dom-event-delegation/main.js

```diff
@@ -0,0 +1,11 @@
+const taskList = document.querySelector('.task-list');
+
+taskList.addEventListener('click', function (e) {
+  const closest = e.target.closest('.task-list-item');
+
+  console.log('Event Target: ', e.target);
+  console.log('Event Tag Name: ', e.target.tagName);
+  console.log('Closest: ', closest);
+
+  closest.remove();
```

Comment:
We should only be removing the closest if the button was the element that was clicked on